Monika After Story/game/script-brbs.rpy

```diff
@@ -110,14 +110,38 @@ label monika_brb_idle:
         show monika at t11
 
         if _return:
-            m 1eua "Oh alright.{w=0.2} {nw}"
-            extend 3hub "Hurry back, I'll be waiting here for you~"
+            $ goodbye = renpy.random.randint(1,2)
+            if goodbye = 1:
+                m 1eua "Oh alright.{w=0.2} {nw}"
+                extend 3hub "Hurry back, I'll be waiting here for you~"
+
+            elif goodbye = 2:
+                m 1eua "Okay."
+                m 3hub "I'll be here when you're done!"
```

Comment:
I don't like this approach, it's very much non-scalable and also inefficient at large scales.

I'd suggest using something like MASQuipList to use label quips as it's cleaner and simpler to expand in future